source/pmc/community-growth.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Community Growth
+tags: ["pmc","committers","community"]
+---
+
+Growing your project community takes work. This document makes practical
+suggestions of how to encourage people to get involved in your project.
+
+## Publish a road map
+
+Volunteers can work on whatever they want, and so we have a tendency to
+not want to tell them what to work on. However, publishing a roadmap, or
+even a wishlist of features or improvements that are desired, can give
+people an idea of where they might be able to get involved.
+
+This also gives a clear sense that the project has ambitions, and is
+going somewhere, rather than that it is stagnating.
```

Comment:
Concretely, it's challenging to maintain a roadmap page on website.

I would propose to "tag" issues with `proposal` or `roadmap` and extract/link this on website.
That's probably the best approach to keep this up to date and accurate.